e2e/paths/caseSubmitted_test.js

```diff
@@ -14,4 +14,10 @@ Scenario('Submitting case', (I, caseViewPage) => {
   I.waitForElement('.tabs');
   I.seeEventSubmissionConfirmation(config.applicationActions.submitCase);
   I.dontSee(caseViewPage.actionsDropdown);
+  caseViewPage.selectTab(caseViewPage.tabs.evidence);
+  I.refreshThePage();
+  I.seeElement('.case-viewer-label');
+  I.seeElement({css : 'ccd-read-document-field>a[href^="http"]'});
```

Comment:
could you not check the contents to see if the information displayed matches what was entered in the test rather than checking that the element tags are present?

Reply:
just to check its actually displaying what you expect it to?

Reply:
I changed to see the case name instead of elements for this